marklogic-data-hub-central/src/test/java/com/marklogic/hub/central/controllers/EntitySearchControllerTest.java

```diff
@@ -214,6 +214,22 @@ void testRowExport() throws Exception {
                 assertRowsAndColumns(newLimit, totalColumns, response);
                 assertTrue(actualRowSet.contains(getHashCode(customer1Info)) || actualRowSet.contains(getHashCode(customer2Info)));
             });
+
+        // Log in as user with role "hub-central-entity-exporter" who is not a "data-hub-operator"
+        loginAsTestUserWithRoles("hub-central-entity-exporter");
+
+        // Export using query document
+        postWithParams(EXPORT_PATH, getRequestParams(limit, json))
+            .andExpect(request().asyncStarted())
+            .andDo(MvcResult::getAsyncResult)
+            .andExpect(status().isOk())
+            .andDo(result -> {
+                String response = result.getResponse().getContentAsString();
+                Set<Integer> actualRowSet = calculateHash(response);
+                assertRowsAndColumns(limit, totalColumns, response);
+                assertTrue(actualRowSet.contains(getHashCode(customer1Info)));
+                assertTrue(actualRowSet.contains(getHashCode(customer2Info)));
+            });
```

Comment:
Instead of adding an extra call to export, we can modify the existing tests to remove "data-hub-operator" and "hub-central-saved-query-user" roles for the first test where we export using query document.

Next, we can add "hub-central-saved-query-user" for login when we test export using queryId which would require for a query to be saved.